packages/spor-alert-react/src/Alert.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Box, BoxProps } from "@vygruppen/spor-layout-react";
+import { As, forwardRef, useMultiStyleConfig } from "@chakra-ui/react";
+
+export type AlertProps = BoxProps & {
+  children: React.ReactNode;
+  variant?: "warning" | "info" | "success" | "error" | "allTransport";
+  title: React.ReactNode;
+  leftIcon: React.ReactNode;
+  rightIcon: React.ReactNode;
+};
+export const Alert = forwardRef<AlertProps, As<any>>(
```

Comment:
Det kan være digg å legge til litt jsdocs for å forklare hva dette er og hvordan man bruker den. Gjerne også med en lenke til dokumentasjonssiden!